helpers.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"net/http"
+	"net/url"
+
+	"github.com/xuri/excelize/v2"
+)
+
+func getCookie(c *http.Client, urlstr, cookieName string) *http.Cookie {
+	urlObj, _ := url.Parse(urlstr)
```

Comment:
check error. if you're sure it will never happen then you can panic if it does

Reply:
I check it and panic if err != nil.